### DISCUSSION TITLE
Semantic Web Flask - for local SPARQL querying

### DISCUSSION THREAD

**Discussion by ghost** [RDFLib/rdflib#1743] [Show and tell]
### [Semantic Web Flask](https://github.com/PaulBrownMagic/SemanticFlask#semantic-web-flask)

This application is for local querying of RDF data. You can drop your files into the "rdf" directory, run the application, open up a browser and SPARQL query your data.

Requirements are all Python3.4+ and can be installed with pipenv. If you don't have pipenv yet, install that first. The last command here activates your virtual environment. The top-level dependencies are Flask, Flask-WTF, Flask-Bootstrap, BeautifulSoup4, RDFLib and requests.

**Comment by white-gecko** [comment]
Looks nice, it reminds me of https://github.com/nareike/adhs
